src/commcare_cloud/ansible/roles/common_installs/tasks/main.yml

```diff
@@ -84,6 +84,7 @@
       - httplib2
       - pexpect
     executable: pip2
+    umask: 0022
```

Comment:
I think that these two installs can be combined by adding:

`virtualenv-clone==0.5.1`

From the docs it seems like that should work (though there are no examples of mixing having versions and not having versions).

Alternatively, it'd probably be good to have versions for these libraries so we can ensure consistent dependencies